### PR TITLE
Fix wrong arrow positions

### PR DESCRIPTION
**Pull request by pierpo**
Solves https://github.com/pierpo/react-archer/issues/93

Now the arrows point to the proper element for the issue #93!

Screenshot from 2020-04-25 12-40-48

### CODE REVIEW

**Comment by jfo84** [comment]
Hey @pierpo I hope you are well. This PR changes the public API since it breaks the style injection and disallows multiple children, which likely has genuine use cases. Maybe this should be pulled back and thought about a bit more.

**Comment by pierpo** [comment]
Hey @jfo84 ! Thanks for following up :blush: 

Yes indeed this is breaking. But the previous behaviour and API didn't make sense.

So, we used to be able to do this:

```jsx
<ArcherElement>
  <div>something</div>
  <div>something</div>
</ArcherElement>
```

Which in fact created the following in the DOM:

```jsx
<div id="fake-element">
  <div>something</div>
  <div>something</div>
</div>
```

And we used `fake-element` to compute the position.

Since the new version, we do this:
```jsx
<ArcherElement>
  <div>something</div>
</ArcherElement>
```

becomes

```jsx
<div>something</div>
```

No new elements are created, we simply take the ref of the given element to compute arrow positions and we know it's exact (it's not the fake position of a parent element that should not exist).

So... now the folling crashes:
```jsx
<ArcherElement>
  {/* Oh no! */}
  <div>something</div>
  <div>something</div>
</ArcherElement>
```

But if you really want to do something like this to replicate the previous behaviour, *you* should wrap the element. You have more control over it and you can properly style it.

```jsx
<ArcherElement>
  {/* This is equivalent to the previous behaviour */}
  <div id="my-own-fake-element">
    <div>something</div>
    <div>something</div>
  </div>
</ArcherElement>
```

Which renders as

```jsx
{/* This is equivalent to the previous behaviour */}
<div id="my-own-fake-element">
  <div>something</div>
  <div>something</div>
</div>
```

**To sum up**, yes the change is breaking in terms of code, but is not breaking in terms of functionality since you can achieve the same as before by adding the element yourself.

Does it make sense? I'd love to have your feedback.

**Comment by jfo84** [comment]
I think it makes sense, but it should probably get a major version like we did before. The change from rendering into a container that we give some API to style is quite different than not having an opinion about any container. I think the usage of ‘React.Children.only’ will cause more problems than you think. The API now does not allow multiple children; that is a significant revision. Allowing the old behavior through a prop sounds like a better way forward. It’s up to you of course, but I still use this library and don’t want it to become unstable.

**Comment by pierpo** [comment]
Thank you for your answer!

I'm not really fond of adding a new prop to use the old behaviour. It's going to add unnecessary noise to both the API and the code.

I'd rather bump to 2.0.0 to notify the breaking change, and require people to comply to this new constraint if they want to have further updates. It's perfectly fine since you can achieve exactly the same as before by adjusting your code.

What do you think?
